bokeh/layouts.py

```diff
@@ -252,7 +264,14 @@ def gridplot(children=None, toolbar_location='left', sizing_mode='fixed', respon
     if toolbar_location:
         if not hasattr(Location, toolbar_location):
             raise ValueError("Invalid value of toolbar_location: %s" % toolbar_location)
-    children = _handle_children(children, *args)
+    if ncols:
+        children = _handle_children(children, wrap=False, *args)
+        has_list = [isinstance(child, list) for child in children]
+        if sum(has_list) > 0:
```

Comment:
`if any(isinstance(child, list) for child in children)` would be a little cleaner / no variable needed